src/main/java/uk/gov/justice/hmpps/prison/repository/sql/AgencyRepositorySql.kt

```diff
@@ -169,4 +169,26 @@ enum class AgencyRepositorySql(val sql: String) {
         ORDER BY USER_DESCRIPTION
     """,
   ),
+
+  GET_AGY_PRISONER_PAY_PROFILE(
```

Comment:
We normally don't write SQL now but use JPA

Reply:
Now using JPA.